tests/ha/test_multi_pod_failure.py

```diff
@@ -2206,6 +2212,231 @@ def test_degraded_deletes_till_kpods_fail(self):
+    @CTFailOn(error_handler)
+    def test_kpods_fail_node_nw_down(self):
+        """
+        Test to Verify degraded IOs after multiple (max K) pods (data and server) failures
+        with network of node hosting them going down.
+        """
+        LOGGER.info("Started: Test to Verify degraded IOs after multiple (max K) pods "
+                    "(data and server) failures with network of node hosting them going down.")
+
+        LOGGER.info("Step 1: Perform WRITE/READ/Verify/DELETEs with variable object sizes.")
+        users = self.mgnt_ops.create_account_users(nusers=1)
+        self.test_prefix = 'test-35788'
+        self.s3_clean = users
+        resp = self.ha_obj.ha_s3_workload_operation(s3userinfo=list(users.values())[0],
+                                                    log_prefix=self.test_prefix)
+        assert_utils.assert_true(resp[0], resp[1])
+        LOGGER.info("Step 1: Performed WRITE/READ/Verify/DELETEs with variable sizes objects.")
+
+        LOGGER.info("Step 2: Delete data and server pods by shutting down network on node they "
+                    "are hosted on.")
+        count = 1
+        data_pod_list = remain_pod_list1 = \
+            self.node_master_list[0].get_all_pods(pod_prefix=const.POD_NAME_PREFIX)
+        server_pod_list = remain_pod_list2 = \
+            self.node_master_list[0].get_all_pods(pod_prefix=const.SERVER_POD_NAME_PREFIX)
+        while count < self.kvalue:
+            pod_data = []
+            ip_data = []
+            resp = self.ha_obj.get_data_pod_no_ha_control(remain_pod_list1,
+                                                          self.node_master_list[0])
+            data_pod_name = resp[0]
+            server_pod_name = resp[1]
+            data_node_fqdn = resp[2]
+            self.pod_name_list.append(data_pod_name)
+            self.pod_name_list.append(server_pod_name)
+            pod_data.append(self.node_master_list[0].get_pod_hostname(pod_name=data_pod_name))
+            LOGGER.info("Get the ip of the host from the node %s", data_node_fqdn)
+            resp = self.ha_obj.get_nw_iface_node_down(host_list=self.host_worker_list,
+                                                      node_list=self.node_worker_list,
+                                                      node_fqdn=data_node_fqdn)
+            node_ip = resp[1]
+            ip_data.append(resp[2]) # node_iface
+            ip_data.append(resp[3]) # new_worker_obj
+            self.node_ip_list.append(node_ip) # node_ip_list
+            assert_utils.assert_true(resp[0], "Node network is still up")
+            LOGGER.info("Step 2: %s Node's network is down.", data_node_fqdn)
+            remain_pod_list1 = list(filter(lambda x: x != data_pod_name, data_pod_list))
+            remain_pod_list2 = list(filter(lambda x: x != server_pod_name, server_pod_list))
+            count += 1
+            self.pod_dict[data_pod_name] = pod_data
+            self.pod_dict[server_pod_name] = pod_data
```

Comment:
Same pod_data for server_pod_name also?

Reply:
Yes as the hostname of the node will be same for both and that is getting used for server pod